src/components/Sidebar.jsx

```diff
@@ -398,8 +402,8 @@ const Sidebar = ({
                     {selected === index ? (
                       <Icon name={upSVG} size="20px" />
                     ) : (
-                      <Icon name={downSVG} size="20px" />
-                    )}
+                        <Icon name={downSVG} size="20px" />
+                      )}
```

Comment:
Not sure if this should be indented like this.. was it checked by prettier? Just double checking to be 100% sure.

Reply:
Yep, vim sometimes messes up with prettier 😞